src/app/index.html

```diff
@@ -2,11 +2,13 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link href="./app.css" rel="stylesheet"/>
```

Comment:
@yshen01 I think we can use import statements to include the css and keep the file and a component directory, per @juangrt 

https://github.com/GRT/react-dev-guide#project-structure--work-in-progress